### ISSUE TITLE
New feature requested for MCC2.1 : New primary event generator

### ISSUE THREAD

**Issue by lemiere** [SuperNEMO-DBD/Falaise#116]
Please, 
Provide double beta decay at high energy : 
Nd150.2nubb_2.5MeV
Ca48.2nubb_3.2MeV

Merci

**Comment by drbenmorgan** [comment]
Closing due to lack of manpower/validation.

**Comment by lemiere** [comment]
So there is no simulation features implemented due to SW dev group failure ? 
@emchauve  : is that a major issue for Snemo Analysis ?

**Comment by emchauve** [comment]
it is not major issue but it is not major work too:

=> just adding this two lines in Falaise/resources/genbb/GenBB.conf 
```
Nd150.2nubb_2.5MeV : Two neutrino double beta decay of Nd-150, 2nubb : DBD/Nd150 :
Ca48.2nubb_3.2MeV : Two neutrino double beta decay of Ca-48, 2nubb : DBD/Nd150 :

```

=> and these 2 paragraphs in  Falaise/resources/genbb/DBDSources.conf 
```
##############################################
[name=" Nd150.2nubb_2.5MeV" type="genbb::wdecay0"]
#@config Two neutrino double beta decay of Nd-150, 2nubb;group=DBD/Nd150
#energy_sum_min=2.5~MeV
decay_type      : string  = "DBD"
decay_isotope   : string  = "Nd150"
decay_dbd_level : integer = 0
decay_dbd_mode  : integer = 4 # 2nubb

#@parameter The energy threshold on the electrons' energy sum
energy_min      : real as energy = 2.5 MeV

##############################################
[name=" Ca48.2nubb_3.2MeV" type="genbb::wdecay0"]
#@config Two neutrino double beta decay of Ca-48, 2nubb;group=DBD/Ca48
#energy_sum_min=3.2~MeV
decay_type      : string  = "DBD"
decay_isotope   : string  = "Ca48"
decay_dbd_level : integer = 0
decay_dbd_mode  : integer = 4 # 2nubb

#@parameter The energy threshold on the electrons' energy sum
energy_min      : real as energy = 3.2 MeV
```

**Comment by drbenmorgan** [comment]
@emchauve, as you have the fix I've assigned to you to implement the Pull Request to fix this. Other than the config changes, we'll also need tests to confirm running in `flsimulate`. Those can be pretty much copied from that of #97 and its fix, so the files to look at are:

https://github.com/SuperNEMO-DBD/Falaise/blob/develop/programs/flsimulate/tests/CMakeLists.txr
https://github.com/SuperNEMO-DBD/Falaise/blob/develop/programs/flsimulate/tests/flsimulate-issue97.conf
https://github.com/SuperNEMO-DBD/Falaise/blob/develop/programs/flsimulate/tests/flsimulate-issue97.profile

**Comment by lemiere** [comment]
Hello,
Is there fresh status about it ?

**Comment by drbenmorgan** [comment]
Closed by #180, can't link and close after a merge!
